cirq/circuits/circuit.py

```diff
@@ -1527,10 +1530,23 @@ def _get_operation_circuit_diagram_info_with_fallback(
 
     return protocols.CircuitDiagramInfo(wire_symbols=symbols)
 
+def _is_exposed_formula(text):
+    return re.match('[a-zA-Z_][a-zA-Z0-9_]*$', text)
+
+
+def _encode(text):
+    return json.JSONEncoder().encode(text)
 
 def _formatted_exponent(info: protocols.CircuitDiagramInfo,
                         args: protocols.CircuitDiagramInfoArgs
                         ) -> Optional[str]:
+
+    if isinstance(info.exponent, sympy.Basic):
+        name = str(info.exponent)
+        return (name
+                if _is_exposed_formula(name)
+                else 'Symbol({})'.format(_encode(name)))
```

Comment:
Good enough for now. May change to just parens `'({})'` later, since it may be a formula instead of a symbol.

Reply:
Done.